doc/reference/peripherals/video.rst

```diff
@@ -0,0 +1,59 @@
+.. _video:
+
+
+Video
+#####
+
+The video subsystem exposes an API to uniformly access video devices.
```

Comment:
"subsystem" bothers me because it's not actually a zephyr/subsys, rather generic video driver api.

Reply:
agree